spec/datadog/tracing/transport/trace_formatter_spec.rb

```diff
@@ -96,7 +120,8 @@
         it do
           is_expected.to have_attributes(
             trace: trace,
-            root_span: root_span
+            root_span: root_span,
+            first_span: first_span
```

Comment:
Minor: This... kinda seems like coupling on an internal detail, e.g. if we ever decide to access the `first_span` directly instead of keeping it as an instance variable, it's kinda weird to have to change the tests?